src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,6 +58,8 @@ public RobotContainer(RobotRunType runtimeType) {
      */
     private void configureButtonBindings(RobotRunType runtimeType) {
         driver.y().onTrue(new InstantCommand(() -> s_Swerve.resetFieldRelativeOffset()));
+        driver.a().onTrue(new InstantCommand(() -> score.runPreScoringMotor(5)));
+        driver.b().onTrue(new InstantCommand(() -> score.runScoringMotor(5)));
```

Comment:
`score.runPreScoringMotor(5)` and `score.runScoringMotor(5)` are already commands so u wouldn't put them in an instant command.